src/internal.c

```diff
@@ -5950,6 +5962,48 @@ int wolfSSL_CTX_IsPrivatePkSet(WOLFSSL_CTX* ctx)
 }
 #endif /* HAVE_PK_CALLBACKS */
 
+static void InitSuites_EitherSide(Suites* suites, ProtocolVersion pv, int keySz,
+        word16 haveRSA, word16 havePSK, word16 haveDH, word16 haveECDSAsig,
```

Comment:
We have 9 parameters when we could have of one parameter that is flags.
Might be too much to change in this PR.

Reply:
I would rather close this PR sooner. If you think this is important, it can be changed in a future PR.